projects/appointment-reminders-and-delius/tech-docs/diag/east-of-england-upw-reminders.d2

```diff
@@ -0,0 +1,77 @@
+# Appointment SMS Flow
+
+direction: down
+
+style { ...@../../../../script/style.style }
+vars { ...@../../../../script/style.vars }
+** { ...@../../../../script/style.all-style }
+
+data: Delius Data {
+        "" {
+            explanation: |md
+            Name, mobile number and appointment date
+            |
+        }
+}
+
+deduplicate: Deduplication  {
+        "" {
+            explanation: |md
+            Where there are multiple appointments for a mobile number\
+            only consider a single reminder
+            |
+        }
+}
+
+validate: Validation {
+        validate_person: Person Validation {
+            explanation: |md
+            - The person is not in custody or on remand
+            - The person is not unlawfully at large
+            - The person is not Limited Access
+            |
+        }
+
+        validate_comms: Communication Validation {
+            explanation: |md
+            - The mobile number is formatted correctly
+            - The person has not opted out of SMS contacts
+            |
+        }
+
+        validate_activity: Activity Validation {
+            explanation: |md
+            - The person has an active unpaid work requirement
+            - There is time remaining on the unpaid work requirement
+            - The appointment does not have an outcome
```

Comment:
Might be worth adding that the unpaid work project type is not ETE-related - the filter is hidden in one of the joins.